src/content/structured/get-started/custom-theme.mdx

```diff
@@ -39,9 +44,148 @@ Alternatively, you can include the `ic-theme` component and set the `color` prop
 <ic-theme color="#5c0949"></ic-theme>
 ```
 
-### Components that use the theme colour
+The components that use the 'brand' colour are:
 
 - [Footer](/components/footer)
 - [Hero](/components/hero)
 - [Side navigation](/components/side-navigation)
 - [Top navigation](/components/top-navigation)
+
+### Theme details
+
+<ComponentDetails component="ic-theme" />
+
+## Dark mode
+
+Each component's variants can be displayed using dark mode colours. This creates better colour contrast with backgrounds, borders and text for when components need to be used on dark backgrounds.
+
+Each component has been given a `theme` prop, taking values of `inherit` (default), `light` and `dark`. By setting the value to either `light` or `dark` at the component level, those components will display dark mode colours, independent of any higher level themes or system settings.
```

Comment:
Not all components have the theme prop. I think this should be communicated as it currently reads as all components have the theme prop